package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "uk.co.compucorp.usermenu",
```

Comment:
Please also add php linters from any of the civicase projects.

Reply:
And the SCSS linters + pre commit hooks for both scss and php.

Reply:
This will take too much time to implement right now, and we only have a few CSS/PHP files. I'll add a ticket for this.